src/accelerate/utils/modeling.py

```diff
@@ -840,10 +843,23 @@ def load_state_dict(checkpoint_file, device_map=None):
             # all weights that haven't defined a device should be loaded on CPU
             device_weights["cpu"].extend([k for k in weight_names if k not in sum(device_weights.values(), [])])
             tensors = {}
+            pbar = is_tqdm_available() and tqdm(
```

Comment:
We do not ever rely on Python magic conversion og objects to bool. This should be a test `if is_tqdm_availbale`with two branches. Also `pbar` should be named `progress_bar` to be explicit.